### DISCUSSION TITLE
[Discussion] Language Support

### DISCUSSION THREAD

**Discussion by josephdadams** [josephdadams/TallyArbiter#238] [Ideas]
I don't fully know what I am asking for when I say this - but what would it take for Tally Arbiter to support multiple languages for the UI other than English? Is this even a desirable feature or just a niche ask?

**Comment by josephdadams** [comment]
Localization. That's the word I was thinking of.

**Comment by hrueger** [comment]
Its not that complicated: https://angular.io/guide/i18n

**Comment by josephdadams** [comment]
Cool. Well I’ll just leave this here until someone asks for it.

**Comment by hrueger** [comment]
Thanks @MatteoGheza for moving this here. Funnily enough I'm working on that right now and will create a PR in a few minutes.
